From 00f320d5f0efdfa24c6ea320fa9b720922954c2d Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua.mayer97@gmail.com>
Date: Sat, 28 Oct 2017 11:14:03 +0200
Subject: [PATCH 1/3] mvebu: clearfog: follow upstream dtb naming

In 712032aa65ae19a613488f8ce956e575cc5e1486, the DTB for Clearfog Pro
has been renamed from armada-388-clearfog to armada-388-clearfog-pro.

Signed-off-by: Josua Mayer <josua.mayer97@gmail.com>
---
 include/configs/clearfog.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/clearfog.h b/include/configs/clearfog.h
index 5061f6c6fd1..51c81e3d97f 100644
--- a/include/configs/clearfog.h
+++ b/include/configs/clearfog.h
@@ -171,7 +171,7 @@
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	RELOCATION_LIMITS_ENV_SETTINGS \
 	LOAD_ADDRESS_ENV_SETTINGS \
-	"fdtfile=" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
+	"fdtfile=armada-388-clearfog-pro.dtb\0" \
 	"console=ttyS0,115200\0" \
 	BOOTENV
 
-- 
2.13.6

